content/intro-to-storybook/react/en/conclusion.md

```diff
@@ -21,6 +21,10 @@ Want to dive deeper? Here are helpful resources.
 
 - [**Visual Testing Handbook**](https://www.chromaticqa.com/book/visual-testing-handbook) dives deep into using Storybook to visual test components. Free 31-page ebook.
 
+- [**Storybook Discord**](https://discord.gg/UUt2PJb) will put you in touch with the Storybook community. Not only offers a means to help you but for you to help the community also.
+
+- [**Storybook blog**](https://medium.com/storybookjs) where you'll read about the latest changes and some techniques to further improve your development workflows with Storybook.
```

Comment:
```suggestion
- [**Storybook blog**](https://medium.com/storybookjs) showcases the latest releases and features to streamline your UI development workflow.
```

Reply:
@domyen thank you for the suggestions. I struggled a bit with these 2 items, to better convey the message. But the change looks real nice simple , straight to the point

Reply:
@domyen marking this as solved per latest commit